src/callbacks/onLayout/augmentOtherControls/yAxisGrouping.js

```diff
@@ -0,0 +1,17 @@
+import { select as d3select } from 'd3';
+
+export default function yAxisGrouping(select, d) {
+    const selected = d3select(select).select('option:checked');
+
+    //Update grouping settings.
```

Comment:
Not urgent, but I threw in an issue to remove the Y-axis grouping control when no groups are specified - #153